testdata/tests_packages/unix-python3.10.txt

```diff
@@ -148,6 +148,7 @@ pure_eval==0.2.2
 py==1.11.0
 pyasn1==0.6.0
 pycowsay==0.0.0.2
+pycowsay==0.0.0.1
```

Comment:
Why add _0.0.1_ to the packages, when _0.0.2_ is already present?

Reply:
I thought `pycowsay` its the default package for testing and example of pipx.

I can change to `nox` but somehow with subprocess I get the output on `stderr` and not `stdout`...
And for `pylint` its more complicated as the output contains multiline check.

I can change to whatever you think its better just tell me :)

Reply:
> I thought `pycowsay` its the default package for testing and example of pipx.

Right, but I think we should make use of available ressources whenever possible.

> I can change to `nox` but somehow with subprocess I get the output on `stderr` and not `stdout`... And for `pylint` its more complicated as the output contains multiline check.

Confirmed, but that isn't a blocker, is it? Besides, have a look at our test packages lists, you could also use `tox` or something else.

Reply:
done, use `ipython` package